hub/core/transform/test_transform.py

```diff
@@ -0,0 +1,96 @@
+from hub.api.dataset import Dataset
+from hub.core.transform.transform import transform
+import numpy as np
+
+
+def fn1(i, mul=1, copy=1):
+    d = {}
+    d["image"] = np.ones((337, 200)) * i * mul
+    d["label"] = np.ones((1,)) * i * mul
+    return [d for _ in range(copy)]
+
+
+def fn2(sample, mul=1, copy=1):
+    d = {}
+    d["image"] = sample["image"] * mul
+    d["label"] = sample["label"] * mul
+    return [d for _ in range(copy)]
+
+
+def fn3(i, mul=1, copy=1):
+    d = {}
+    d["image"] = np.ones((4310, 2187)) * i * mul
+    d["label"] = np.ones((13,)) * i * mul
+    return [d for _ in range(copy)]
+
+
+def test_transform_hub_dataset(local_storage):
+    with Dataset("./test/transform_hub_in") as data_in:
+        data_in.create_tensor("image")
+        data_in.create_tensor("label")
+        for i in range(100):
+            data_in.image.append(i * np.ones((100, 100)))
+            data_in.label.append(i * np.ones((1,)))
+    data_in = Dataset("./test/transform_hub_in")
+    ds_out = Dataset(storage=local_storage)
```

Comment:
for these 2 you can derive from the fixture (please please do, the fixture auto-populates the paths / storages)

Reply:
Is there an easy way to use 2 ds fixtures in a single fn?

Reply:
Added to the backlog as discussed.